schema.ts

```diff
@@ -870,6 +870,10 @@ export interface PropertyIdentifierDetails {
    * If set to '', the label will be omitted.
    */
   label?: string;
+  /**
+   * An optional placeholder value, in case the property value is null or undefined.
```

Comment:
I think you should clarify when this placeholder is used. Only when the label is also shown (subtitles) or anytime the value is shown (card title, snippet, etc)? Is the placeholder value returned when the property is accessed via CFL?

Reply:
Yeah I think we can do card title, subtitle, and snippet

Reply:
@erickoledadevrel updated this part

Reply:
Looks good!